### PR TITLE
make Redis installation optional

### LINKED ISSUE #67: make Redis installation optional
If we disable redis installation, the charts render unusable. In order to make this work, some changes need to be made to the templates.

If redis installation disabled, there will be a missing key error:

Rendered `rasa-deployments.yaml`:

```
[...]
        - name: "REDIS_PASSWORD"
          valueFrom:
            secretKeyRef:
              name: "rasa-x-redis"
              key: 
[...]
```

### PR DESCRIPTION
**Pull request by loomsen**
This PR fixes #67

### CODE REVIEW

**Comment by sara-tagger** [comment]
Thanks for submitting a pull request 🚀 @koaning will take a look at it as soon as possible ✨

File: charts/rasa-x/templates/rasa-config-files-configmap.yaml [hunk 1]
```diff
@@ -37,9 +37,11 @@ data:
...
       url: "http://{{ include "rasa-x.fullname" . }}-app:{{ template "rasa-x.custom-actions.port" . }}{{ .Values.app.endpoints.actionEndpointUrl }}"
       token:  ""
+    {{- if $.Values.redis.install }}
     lock_store:
       type: "redis"
       url: {{ template "rasa-x.redis.host" . }}
       port: {{ default 6379 .Values.redis.redisPort }}
       password: ${REDIS_PASSWORD}
       db: {{ default "1" .Values.rasa.lockStoreDatabase }}
+    {{- end }}
```
Comment (tczekajlo) [inline charts/rasa-x/templates/rasa-config-files-configmap.yaml:47]: No newline at the end of the file.

**Review by tczekajlo — COMMENTED** [review]
@loomsen Thanks a lot for your PR, I've left one comment :)

File: charts/rasa-x/templates/rasa-config-files-configmap.yaml (hunk 1 above)
Comment (loomsen) [reply charts/rasa-x/templates/rasa-config-files-configmap.yaml]: There we go :+1:

**Review by tczekajlo — APPROVED** [review]
LGTM

**Pull request merged**